source/delegate-factory/docsTemplate/contract.hbs

```diff
@@ -0,0 +1,70 @@
+## <span id="{{anchor}}"></span> `{{name}}`
+
+ Delegate is a smart contract that sends orders based on rules. View the code on GitHub.
+
+`{{{natspec.title}}}`
+
+{{{natspec.userdoc}}}
+
+{{{natspec.devdoc}}}
+
+### Functions
+
+{{#ownFunctions}}
+### <span id="{{anchor}}"></span> `{{name}}({{args}}){{#if outputs}} → {{outputs}}{{/if}}` ({{visibility}})
+
+{{{natspec.userdoc}}}
+
+{{{natspec.devdoc}}}
+{{#if natspec.params}}
+| Param             | Description       |
+{{#each natspec.params}}
+{{#if @first}}
+| ----------------- | :------------    |
+{{/if}}|`{{param}}`|{{description~}}{{#raw}}|{{/raw}}{{/each}}{{/if}}
+
+{{#if natspec.returns}}
+|  Param    | Description   |
+{{#each natspec.returns}}
+{{#if @first}}
+|   ---------| :------------ |
+{{/if}}| `{{param}}`|  {{description~}}{{#raw}}|{{/raw}}{{/each}}{{/if}}
+
+{{/ownFunctions}}
+
+## Price Calculations
```

Comment:
is there a reason we need price calculations in the delegate-factory docs when they're already in the delegate docs?

Reply:
will remove.